page/index.html

```diff
@@ -1,99 +1,158 @@
 		<p>A noisy, underplotted, and overlong special effects extravanganza that lacks a human touch.</p>
-		<p>ROTTEN TOMATO RATING</p>
-		<p>20%</p>
-		<p>WORLD PREMIER</p>
-		<p>DEN HAAG THEATRE</br>THE NETHERLANDS</p>
-		<p>JUNE 23, 2011</p>
+		<div class="dashedbordertop">
+			<div class="rottentomato">
+				<p>ROTTEN TOMATO</br><span class="rottenrating">RATING</span></p>
+				<p class="rottenpercentage">20%</p>
+				<div class="rottenratingbar"><span></span></div>
+				</div>
+				<div class="dashedborderbottom"></div>
+				</div>
+				<div class="worldpremiere">
+					<div class="barworldpremier">
+						<div class="barworld"></div>
+						<p>WORLD PREMIER</p>
+						<div class="barworld"></div>
+					</div>
+					<p>DEN HAAG THEATRE</p>
+					<p>THE NETHERLANDS</p>
+					<p>JUNE 23, 2011</p>
+					<div class="circlewhite"></div>
+				</div>
 	</article>
-	<article>
+	<article class="overflow">
 		<figure>
 			<img src="../images/water/text.png" alt="">
 		</figure>
-		<section>On the night of Aug 8, 1975, a line of people frantically piled sandbags atop Henan Province's <strong>Banqiao Dam</strong> while being battered by the worst storm ever recorded in the region.  They were in a race with the rapidly rising Ru River to save the dam and the millions of people that lay sleeping downstream.  It was a race they were about to lose.</section>
-		<section>Just after the 1:00 am, the sky cleared and stars emerged from behind the storm clouds.  There was an eerie calm as someone yelled, "The water level is going down! The flood is retreating!"</section>
-		<section>There was little chance to enjoy that calm.  One survivor recalled that a few seconds later it "sounded like the sky was collapsing and the earth was cracking." The equivalent of 280,000 Olympic-sized swimming pools burst through the crumbling dam, taking with it entire towns and as many as 171,000 lives.</section>
+		<p class="overflow-one">On the night of Aug 8, 1975, a line of people frantically piled sandbags atop Henan Province's <span>Banqiao Dam</span> while being battered by the worst storm ever recorded in the region. They were in a race with the rapidly rising Ru River to save the dam and the millions of people that lay sleeping downstream. It was a race they were about to lose.</p>
+		<p class="overflow-two">Just after the 1:00 am, the sky cleared and stars emerged from behind the storm clouds. There was an eerie calm as someone yelled, "The water level is going down! The flood is retreating!"</p>
+		<p class="overflow-three">There was little chance to enjoy that calm. One survivor recalled that a few seconds later it "sounded like the sky was collapsing and the earth was cracking." The equivalent of 280,000 Olympic-sized swimming pools burst through the crumbling dam, taking with it entire towns and as many as 171,000 lives.</p>
 	</article>
+	<form name="registration" class="registrationform">
+		<h1> Registration Form: </h1>
+		<label for="firstname">First Name:</label>
```

Comment:
The `for` attribute of the label element must refer to a non-hidden form control.